### ISSUE TITLE
Failing handshake due to unexpected packet format

### ISSUE THREAD

**Issue by Jirubizu** [midwan/MB_SubSonic#62]
When trying to connect to an airsonic server being run from ultra.cc I get the following error.
![image](https://chibi.jirubizu.cc/dy6moc.png)

**Comment by midwan** [comment]
@Jirubizu 
I can't recreate that in a standard environment with Airsonic here...
I've tried setting up Airsonic under Docker, using this image: https://github.com/linuxserver/docker-airsonic
Using the latest version of MB (currently 3.4.7805 P) and the latest version of the plugin (v2.32).

It seems that the network configuration for your Airsonic server is somehow different than the expected. Is it behind some proxy/firewall? Either way, the plugin did not get the expected HTTP/HTTPS response from the server.

**Comment by Jirubizu** [comment]
I am running my instance of airsonic on ultraseedbox, I get the following information about accessing airsonic from the site
![image](https://user-images.githubusercontent.com/37895773/128614409-badd2e73-e67e-49e8-82e1-7379fe8d1369.png)
This is the config that I am trying to input
![image](https://user-images.githubusercontent.com/37895773/128614430-5dcc878c-83ed-4d1d-af53-ddee722a4821.png)
I am replacing the values in {} with the corresponding stuff from the previous screenshot but nothing seems to work :/

**Comment by Jirubizu** [comment]
@midwan
Forgot to mention but I have tried both token and hex password method but nothing worked

**Comment by midwan** [comment]
It's hard to say what's wrong exactly, but based on the error message, something went wrong during the initial Ping command.
Perhaps you can test this yourself and see what you get back, from a browser.
The Subsonic API documentation is here: http://www.subsonic.org/pages/api.jsp

You can see some basic examples of the Ping endpoint as well. What happens if you try that from your browser?

**Comment by Jirubizu** [comment]
So it seems to work when doing the API calls. Am I possibly filling out the dialog from the plugin wrong?
![Screenshot](https://chibi.jirubizu.cc/zgukrl.png)

**Comment by Jirubizu** [comment]
Even using 1.13.0 as the version. I still get an OK response @midwan

**Comment by Jirubizu** [comment]
Ok so, I ended up getting it working. This was what was wrong. I was providing a port when I wasn't using an API and using the HTTPS protocol. so removing the port helped. Secondly, I got confused by the path. I assumed it meant the default media path however, I had to provide the URI path to the exact location of the WebUI. after that, it worked fine. Sorry for the hassle I caused. I should have read the source code and understood how the plugin parses stuff.

![image](https://user-images.githubusercontent.com/37895773/128630854-eaf7f00a-c5d5-4c21-8a8a-e34113785612.png)
